File: Php/app/Urls/CashgameUnpublishUrlModel.php
<?php
namespace app\Urls{

	use app\RouteFormats;
	use app\Urls\BaseClasses\CashgameUrlModel;
	use entities\Cashgame;
	use entities\Homegame;

	class CashgameUnpublishUrlModel extends CashgameUrlModel{

		public function __construct(Homegame $homegame, Cashgame $cashgame){
			parent::__construct(RouteFormats::cashgameUnpublish, $homegame, $cashgame);
		}

	}

}

File: Web/Models/UrlModels/CashgameUnpublishUrlModel.cs
using Core.Classes;
using Web.Routing;

namespace Web.Models.UrlModels{
    public class CashgameUnpublishUrlModel : CashgameUrlModel{

		public CashgameUnpublishUrlModel(Homegame homegame, Cashgame cashgame)
            : base(RouteFormats.CashgameUnpublish, homegame, cashgame)
        {
		}

	}

}